app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  after_create :update_likes_counter
+  after_destroy :update_likes_counter
+
+  belongs_to :user
+  belongs_to :post, counter_cache: :likes_counter
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
### Optional
- [ ] Also for your likes count, you can also use `post.increment!(:likes_counter)` to update likes count